.github/workflows/workflow.yml

```diff
@@ -197,3 +197,31 @@ jobs:
           echo "${{ secrets.FRANCE_API_DEPLOY_KEY }}" > /home/runner/.ssh/france_api_deploy
           chmod 600 /home/runner/.ssh/france_api_deploy
           ssh -i /home/runner/.ssh/france_api_deploy -o StrictHostKeyChecking=no deploy-api@fr.openfisca.org
+
+  check-api-version:
+    runs-on: ubuntu-latest
+    needs: [ deploy ]
+    outputs:
+      version-match: ${{ steps.check-api-version.outputs.version-match }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Cache build
+        id: restore-build
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-build-${{ hashFiles('setup.py') }}
```

Comment:
It seems like the used packages are all system packages, why do we need the whole build? 🤔

Reply:
For the `requests` module. I did not want to pollute with pip install :)